dev/scripts/src/alluxio.org/build-distribution/cmd/root.go

```diff
@@ -21,6 +21,7 @@ var (
 	customUfsModuleFlag string
 	debugFlag           bool
 	ufsModulesFlag      string
+	includedLibJarsFlag string
```

Comment:
move up a line to maintain alphabetical order

Reply:
Thanks so much for the suggessions.